packages/pds/src/db/migrations/20230804T224916736Z-action-duration.ts

```diff
@@ -0,0 +1,27 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await Promise.all([
+    db.schema
+      .alterTable('moderation_action')
+      .addColumn('actionDurationInHours', 'integer')
+      .execute(),
+    db.schema
+      .alterTable('moderation_action')
+      .addColumn('actionExpiresAt', 'varchar')
+      .execute(),
+  ])
```

Comment:
How would you feel about dropping the `action*` prefix here?  I think `durationInHours` and `expiresAt` would be clear enough, and maybe a bit more natural to write e.g. `action.expiresAt`.  I don't think we have similar prefixes elsewhere, so it could be good for consistency too.

Reply:
agreed!